org-roam.el

```diff
@@ -1378,7 +1379,8 @@ If DESCRIPTION is provided, use this as the link label.  See
                                                                :link-description link-description
                                                                :capture-fn 'org-roam-insert))
         (org-roam--with-template-error 'org-roam-capture-templates
-          (org-roam-capture--capture))))))
+          (org-roam-capture--capture))))
+    res))
```

Comment:
if the user didn't select existing note, the value will be `nil`. We could return `title-with-tags`, but the problem is that `org-roam-insert` finishes before capture process is done. So it's totally fine to have `nil`.